src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -388,7 +388,8 @@ private void processContingenciesBreakingConnectivity(ConnectivityBreakAnalysis.
                 connectivityAnalysisResult.getElementsToReconnect(), resultWriter, reportNode, partialDisabledBranches, rhsChanged);
     }
 
-    protected void cleanContingencies(LfNetwork lfNetwork, List<PropagatedContingency> contingencies) {
+    // TODO : move this method in PropagatedContingencies in another PR
+    public static void cleanContingencies(LfNetwork lfNetwork, List<PropagatedContingency> contingencies) {
```

Comment:
Not allowed, to be fixed this that PR or in a PR before.

Reply:
@p-arvy what about this issue?

Reply:
Thank for getting back to this @annetill, I opened a PR dedicated to this issue: https://github.com/powsybl/powsybl-open-loadflow/pull/1081